src/EditorFeatures/Test2/FindReferences/FindReferencesTests.ParameterSymbol.vb

```diff
@@ -932,7 +932,6 @@ using System;
 class Goo(int x, int {|Definition:$$y|})
 {
     public int y { get; } = [|y|];
-    public int z { get; } => [|y|];
```

Comment:
curious about the removals here?  was that intentional/needed?

Reply:
> curious about the removals here?  was that intentional/needed?

I added dedicated test for this scenario. The property `y` declared above shadows the parameter.

Reply:
ah, gotcha.  IMO, it would be better to leave the line in, but take off the `[| |]` around it (to make it clear that that 'y' is *not* considered a ref.  But up to you.

Reply:
Put the lines back as suggested